client/context/context.go

```diff
@@ -34,6 +28,7 @@ type CLIContext struct {
 	Codec         *codec.Codec
 	AccDecoder    auth.AccountDecoder
 	Client        rpcclient.Client
+	TxBldr        authtxb.TxBuilder
```

Comment:
Would be interested to hear other thoughts from the team @cosmos/cosmossdk

Reply:
@alessio's points make sense to me - are there counterarguments?

Reply:
Totally agree here. I also think the `fromName` and `fromAddress` fields (and corresponding logic to get these values) should be moved into the `TxBuilder`.

Reply:
In all honesti, I think we'd better keep `CLIContext` and `TxBuilder` separate to avoid establishing any unnecessary dependency link between the two. For sake of caller's convenience though, I'm OK with `TxBuilder` to encapsulate `.CLIContext`.

Reply:
Yeahhh, I really don't see a big reason for this PR then imho. Sure, the CLI context can be cleaned up a bit (some of which I'll be doing in #3396)